### PR TITLE
Block discord.com invites

### PR DESCRIPTION
**Pull request by AnonGuy**

### CODE REVIEW

File: cdbot/constants.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ class Exchange:
 ROOT_ROLE_ID = int(environ.get("ROOT_MEMBERS_ID", "450113490590629888"))
 SUDO_ROLE_ID = int(environ.get("SUDO_MEMBERS_ID", "450113682542952451"))
 ADMIN_ROLES = ("Root", "Sudo")
-BANNED_DOMAINS = ["discord.gg"]
```
Comment (thebeanogamer) [inline cdbot/constants.py:116]: I don't think we actually ever call this.

Comment (AnonGuy) [reply cdbot/constants.py:116]: true

Comment (AnonGuy) [reply cdbot/constants.py:116]: I changed it, checked for usage and figured there's little point changing it back

Comment (thebeanogamer) [reply cdbot/constants.py:116]: Yeah, may as well

**Pull request merged**